build-tools/packages/build-cli/src/commands/generate/buildVersion.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable camelcase */
```

Comment:
This is for `child_process` right? Would it be better to just import it as `ChildProcess` instead?